components/script_bindings/webidls/CSSNestedDeclarations.webidl

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * The origin of this IDL file is
+ * https://drafts.csswg.org/css-nesting-1/#the-cssnestrule
+ */
+
+[Exposed=Window]
+interface CSSNestedDeclarations : CSSRule {
+  // CSSStyleDeclaration instead of CSSStyleProperties for now, see bug 1919582.
```

Comment:
Do we depend on this firefox bug[^1] because stylo doesn't support the functionality needed for `CSSStyleProperties`?

[^1]: https://bugzilla.mozilla.org/show_bug.cgi?id=1919582

Reply:
Pointed to #36260 instead